### PR TITLE
Add warning about latest tag

### PR DESCRIPTION
**Pull request by MaxymVlasov**
Suggestion based on
https://github.com/gruntwork-io/infrastructure-as-code-training/pull/24#issuecomment-787908656

### CODE REVIEW

File: docker-example/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# This is example, do not use `latest` tag in real life, please.
+# Details: https://vsupalov.com/docker-latest-tag/
```
Comment (brikis98) [inline docker-example/Dockerfile:2]:
```suggestion
# In this example, we use the 'latest' tag for simplicity. In production code, we recommend pinning to a more specific version.
# Details: https://vsupalov.com/docker-latest-tag/
```

**Review by brikis98 — COMMENTED** [review]
Thanks for the PR!

**Comment by MaxymVlasov** [comment]
Oh, looks like I miss your suggestion.
Done.

**Review by brikis98 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**